Makefile

```diff
@@ -1,7 +0,0 @@
-SUBDIRS := BusyCoq/. CoqBB5/.
-
```

Comment:
couldn't you just add `CoqBB5/.: BusyCoq/.` ?

Reply:
The issue is that when doing `make -j 13` at the root, it will start trying to compile CoqBB5 files before BusyCoq is done and fail because of dependencies are not compiled and not try again. Hence, only BusyCoq get compiled.

Reply:
Given that BusyCoq only takes about 7 minutes to compile I feel like `cd BusyCoq && make -j 13 && cd ../CoqBB5 && make -j 13` is a good compromise as I'm not fluent enough with Makefile to find the right tricks to use to make it work as intended.

Reply:
> it will start trying to compile CoqBB5 files before BusyCoq 

not if you add what I said

Reply:
Based on your comments, which I'm not 100% sure how to interpret, I tried the following:

```Makefile
SUBDIRS := CoqBB5/.: BusyCoq/.

all: $(SUBDIRS)
$(SUBDIRS):
		$(MAKE) -C $@

.PHONY: all $(SUBDIRS)
```

This fails with `Makefile:3: *** target pattern contains no `%'.  Stop.`

Could you provide an entire (tested) Makefile?

Thank you very much!

Reply:
take the old makefile and add `CoqBB5/.: BusyCoq/.` as a separate line (like the .PHONY line)

PS using `CoqBB5` instead of `CoqBB5/.` would look nicer

~~~gmake
SUBDIRS := BusyCoq CoqBB5

CoqBB5: BusyCoq

all: $(SUBDIRS)
$(SUBDIRS):
		$(MAKE) -C $@

.PHONY: all $(SUBDIRS)
~~~

Reply:
actually that probably doesn't work because the `$(SUBDIRS):` rule means make expects to make all subdirs at once
so
~~~make
SUBDIRS := BusyCoq CoqBB5

all: $(SUBDIRS)

BudyCoq:
		$(MAKE) -C $@

CoqBB5: BusyCoq
		$(MAKE) -C $@

.PHONY: all $(SUBDIRS)
~~~

Reply:
It worked!! Thank you, see ee7e25acef2b5260d88c20112b423b9d5e57899f :)